repo/packages/E/etcd/0/config.json

```diff
@@ -0,0 +1,123 @@
+{
+  "properties": {
+    "etcd": {
+      "description": "etcd specific configuration properties",
+      "properties": {
+        "docker-image": {
+          "default": "mesosphere/etcd-mesos:latest",
```

Comment:
it's nice to version docker image tags at some point. i've seen timestamps or semantic versions. not sure what mesosphere standards are here